setup.py

```diff
@@ -22,26 +22,27 @@ def hidapi_src(platform):
 
 if sys.platform.startswith("linux"):
     modules = []
-    if "--without-libusb" in sys.argv:
-        sys.argv.remove("--without-libusb")
-        hidraw_module = "hid"
-    else:
+    if "--with-libusb" in sys.argv:
+        sys.argv.remove("--with-libusb")
         hidraw_module = "hidraw"
         libs = ["usb-1.0", "udev", "rt"]
+        src2 = src.copy()
+        src = ["hidraw.pyx", "chid.pxd"]
```

Comment:
I think you can drop `src2` if you are overwriting `src`?

Reply:
Yes (although not at that point).  I'm not happy with the control flow mess around this place, will clean it up.

I also just realized I need to explicitly accept `--without-libusb` even if it's now the defautl behavior, otherwise users will get usage errors. This is particularly relevant for Linux distro packages (e.g. ArchLinux).